sandbox

```diff
@@ -169,6 +169,25 @@ sandbox () {
     docker-compose -f "$SANDBOX_DIR/docker-compose.yml" "$@"
   }
 
+  dc_no_cache () {
+    docker-compose build --no-cache
```

Comment:
You should always use `dc` instead of `docker-compose` because `dc` specify the docker-compose file